packages/mui-material-next/src/styles/state.ts

```diff
@@ -1,4 +1,6 @@
-const mdSysState = {
+import { MD3State } from './Theme.types';
```

Comment:
The file name was not following the convention for matching the theme's key, which is singular.